Mathlib/Algebra/Category/Grp/Images.lean

```diff
@@ -86,14 +86,14 @@ theorem image.lift_fac (F' : MonoFactorisation f) : image.lift F' ≫ F'.m = ima
 end
 
 /-- the factorisation of any morphism in `AddCommGrp` through a mono. -/
-def monoFactorisation : MonoFactorisation f where
+def MonoFactorization : MonoFactorization f where
```

Comment:
There are a few more definitions, like this one, for which a `deprecated` alias would be welcome.

Reply:
My bad, fixed the capitalization that was accidentally changed. I also added aliases to all definitions.